misc/systemd/ipfs.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+description=ipfs p2p daemon
+After=network.target
+Requires=network.target
```

Comment:
A ipfs daemon doesn't actually need a network to start and the user can interact with several functions of ipfs without a network connection. Granted most people will want a network connection but part of the issue with putting examples in the repo is that people will take them as how things 'should' or 'must' work and that isn't necessarily the case.

Reply:
Now you're just being pedantic.  There is zero point to using the ipfs daemon without a network connection; all of the local-only options work fine without a daemon.  You certainly don't want the ipfs daemon starting up before the network as it discovers its network interfaces on startup.

Reply:
The commands do work offline but, in this case, they'll only work if you happen to be the `ipfs` user or root.

Side note: Units generally should *not* use `Requires=network.target`, only `After=network.target` (see the documentation in `man 7 systemd.special`).